.github/workflows/test-suite.yml

```diff
@@ -116,6 +116,7 @@ jobs:
         php-version:
           - "8.1"
           - "8.2"
+          - "8.3"
```

Comment:
Before that, all php constraints should be set to more precise `~8.1.10 || ~8.2.0 || ~8.3.0`

Reply:
Moved out to next PR.

Reply:
It should be merged before or with this one since it's fixing bug with PHP version constraint

Reply:
Let's merge it before this one, here it goes: https://github.com/aeon-php/aeon/pull/116